content/source/docs/enterprise/api/_template.md

```diff
@@ -21,6 +16,13 @@ Parameter            | Description
 
 If further explanation of this method is needed beyond its title, write it here, after the parameter list. -->
 
+Result  | Status and response
+--------|------------------------
+Success | HTTP 200 and a JSON API document (`type: "somethings"`).
```

Comment:
Oh I meant `Status` and `Response`. 😜 

Status | Response
------ | ----------
`200` | JSON API document (`type: "somethings"`)
`204` | No content
`400` | JSON API document

Maybe the codes can link to official documentation too?

Reply:
https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/commit/%7Bnode%7D/statuses/build

Here you can see multiple status codes in place.

Reply:
@dylanegan augh, sorry! 😭 Wellllll it's merged, so, do you want to do a second PR with your proposal? Actually, maybe using real info from an endpoint would be more illustrative.

Reply:
@nfagerlund what do you mean by real info? Like a response? It might be good, but then we'd drop the `Sample Response` part in favour of this? And have it as `Sample Responses`?

Reply:
@dylanegan oh, no, I meant, like, the real status codes from one of our endpoints. Like, whichever one you said was using 409?

Reply:
@nfagerlund oh yeah, heh.

`POST /organizations/:organization_name/teams`

Status | Response | Reason(s)
---- | ------------ | -------
[200](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200) | [JSON API document](https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents) | Successfully created a team
[400](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400) | [JSON API error object](http://jsonapi.org/format/#error-objects) | Invalid `include` parameter
[404](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404) | [JSON API error object](http://jsonapi.org/format/#error-objects) | Organization not found, user is unauthorized to perform action
[422](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422) | [JSON API error object](http://jsonapi.org/format/#error-objects) | Validation errors
[500](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500) | [JSON API error object](http://jsonapi.org/format/#error-objects) | Failure during team creation